tasmota/tasmota_template.h

```diff
@@ -172,6 +172,7 @@ enum UserSelectablePins {
   GPIO_MCP2515_CS,                     // MCP2515 Chip Select
   GPIO_HRG15_TX, GPIO_HRG15_RX,        // Hydreon RG-15 rain sensor serial interface
   GPIO_VINDRIKTNING_RX,                // IKEA VINDRIKTNING Serial interface
+  GPIO_BL6523_TX, GPIO_BL6523_RX,                      // BL6523 based Watt meter Serial interface
```

Comment:
This is wrong location
All new GPIO _MUST_ be added at the end of the list, right before GIO_SENSOR_END